content/_index.md

```diff
@@ -15,7 +15,14 @@ container: "container-fluid"
 header_wrapper_class: "featured-jumbotron-home"
 ---
 {{< home/powered-by >}}
-{{< home/highlights >}}
+
+{{< home/highlights
```

Comment:
Is this a component that we would want to re-use on other sites?
If so, we should contribute to the base theme as I did with the search page: https://gitlab.eclipse.org/eclipsefdn/it/webdev/hugo-solstice-theme/-/merge_requests/246

Reply:
Yes, I'll try to make that as generic as possible. 

This is a featured story block that is pulling multiple items. I could create a shortcode called "featured-story-multiple" on hugo theme. I don't think we should modify the existing featured-story shortcode since the html is not the same in the 2 cases.

What do you think?